meinberlin/apps/notifications/signals.py

```diff
@@ -18,7 +18,7 @@
 @receiver(signals.post_save, sender=Action)
 def send_notifications(instance, created, **kwargs):
     if created:
-        if settings.TEST:
+        if hasattr(settings, "TEST") and settings.TEST:
             send_action_notifications.delay(instance.pk)
```

Comment:
thanks. Also if settings has no attribute `TEST`, and we are in testing enviroment tests will fail because of the atomic transaction of the `delay_on_commit`. Maybe you can add a docstring to indicate that too?